Jellyfin.Plugin.NextPVR/LiveTvService.cs

```diff
@@ -457,12 +457,13 @@ public async Task CancelSeriesTimerAsync(string timerId, CancellationToken cance
         _logger.LogInformation("[NextPVR] Cancelled Recording for recordingId: {TimerId}", timerId);
     }
 
-    public Task<List<MediaSourceInfo>> GetChannelStreamMediaSources(string channelId, CancellationToken cancellationToken)
+    public async Task<List<MediaSourceInfo>> GetChannelStreamMediaSources(string channelId, CancellationToken cancellationToken)
     {
-        throw new NotImplementedException();
+        var source = await GetChannelStream(channelId, string.Empty, cancellationToken);
+        return new List<MediaSourceInfo>() { source };
     }
 
-    public Task<MediaSourceInfo> GetChannelStream(string channelId, string streamId, CancellationToken cancellationToken)
+    public async Task<MediaSourceInfo> GetChannelStream(string channelId, string streamId, CancellationToken cancellationToken)
```

Comment:
```suggestion
    public Task<MediaSourceInfo> GetChannelStream(string channelId, string streamId, CancellationToken cancellationToken)
```

Reply:
I tried those changes earlier  (which is why I tried the TVHeadend way) but this doesn't give me live TV either.  See my pastebin on developer element.io

Reply:
Sadly that doesn't really help, and I have no way to test this plugin.

Reply:
If you have IPTV or any http URL is is easy to setup, I could set up a remote connection if needed.  In any case I can't explain why the Jellyfin backend has started calling a new function that was never called before and why simply returning the value from the old function in a List<> don't work.  The plugin is really quite stupid. Everything was fine in alpha2 the last time I tested.

Reply:
@anthonylavado since @crobibero says the the new Live TV call in 10.8 cannot be debugged and there are no instructions on how the addon is supposed to adapt to the change should the addon be withdrawn?

Reply:
@crobibero I don't know if it helps but this url from the browser jellyfin server is generating a 500 error (I forced h264 and aac just to see if that was the issue)  I don't 

```
GET	http://localhost:8096/videos/7ac50a58-a91e-9b1e-3843-361be12dc610/master.m3u8?DeviceId=TW96aWxsYS81LjAgKFdpbmRvd3MgTlQgMTAuMDsgV2luNjQ7IHg2NDsgcnY6NjcuMCkgR2Vja28vMjAxMDAxMDEgRmlyZWZveC82Ny4wfDE1NjA5MDgwNDAzOTQ1&MediaSourceId=1&VideoCodec=h264&AudioCodec=aac&AudioStreamIndex=-1&VideoBitrate=559360000&AudioBitrate=640000&MaxWidth=3840&PlaySessionId=6be0e61b0e7d4f79941876b003cfcb3f&api_key=2ca1957adbfc4053b71ebb34b4175df7&TranscodingMaxAudioChannels=8&RequireAvc=false&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=VideoCodecNotSupported
```

The plugin is called from here https://github.com/jellyfin/jellyfin/blob/master/Emby.Server.Implementations/LiveTv/LiveTvManager.cs#L270 and AFAIK its seems it returns the same values in the List that that the old call returned.  I don't see how to trap the 500 error.

Reply:
Changes made in update